chemprop/cli/train.py

```diff
@@ -1028,3 +1058,61 @@ def main(args):
     logging.basicConfig(stream=sys.stdout, level=logging.DEBUG, force=True)
     args = parser.parse_args()
     TrainSubcommand.func(args)
+
+
+def summarize(task_type: str, dataset: MoleculeDataset) -> tuple[list, list]:
+    if task_type in ["regression", "regression-mve", "regression-evidential"]:
+        y = np.array([datapoint.y[0] for datapoint in dataset.data])
```

Comment:
I think as you address my other comment about reporting statistics per task in multi-task, you can incorporate this change:
```suggestion
        y = dataset.Y
```